### DISCUSSION TITLE
Welcome to PCF Builder Discussions!

### DISCUSSION THREAD

**Discussion by Power-Maverick** [Power-Maverick/PCF-CustomControlBuilder#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, click on New Discussion button on the "Discussion" tab.

**Comment by Freeetje** [comment]
Hi,

First of all thank you for the fantastic tool! I use it for all my "PCF Controls".

But I do have a question about the "Increment Version"-option. I want to have 4 numbers in de version (f.e. 1.0.0.3) instead of 3 numbers (f.e. 1.0.3). 

![image](https://user-images.githubusercontent.com/80949930/111755044-f6aef600-8898-11eb-9cf5-ff84589803b6.png)

I changed the version "Solution"-xml, but then it overwrites the number again with 1.0.1. I also tried to change it in the solution itself, with no result...

Is there a solution to do this?

Kind Regards,
Frederic

**Reply by Power-Maverick** [reply]
I see the issue and unfortunately there is no way to use 4 level schematic versioning within the tool but I'll make a note of it for improvement in the next release. In meantime if you want to log it as a bug please feel free. Thank you.
